scripts/metrics/ctest_metrics_xml_to_csv.py

```diff
@@ -70,8 +79,21 @@ def parse_args():
     )
     parser.add_argument(
         "--csv-file", action="store", default=_get_default_csv_filename(),
-        help=f"The directory and file name for the csv to be saved (defaults to YYYY_MM_DD)."
+        help=f"The directory and file name for the csv to be saved (defaults to YYYY_MM_DD_HH_mm)."
```

Comment:
Would be nice to have a strict contract of how folders are structured in S3, to share between upload and download/querying. If it doesn't get added in code, be sure to document it after iteration is done.

Reply:
Folder structure contract should be held by unit tests. Unsure if the download/querying has tests, but will cut a task if not.